src/main.rs

```diff
@@ -355,6 +357,35 @@ async fn main() {
     }
 }
 
+fn respawn_swaps(
+    db: Arc<Database>,
+    bitcoin_wallet: Arc<bitcoin_wallet::Wallet>,
+    ethereum_wallet: Arc<ethereum_wallet::Wallet>,
+    bitcoin_connector: Arc<comit::btsieve::bitcoin::BitcoindConnector>,
+    ethereum_connector: Arc<comit::btsieve::ethereum::Web3Connector>,
+    swap_execution_finished_sender: Sender<FinishedSwap>,
+) -> anyhow::Result<()> {
+    // Assumption: all swaps in the database are active swaps
```

Comment:
Hum, good one. So we need a call to delete the swap from the DB after it's done and we saved it in the csv file. Can you add this call to this PR?

Reply:
Done with https://github.com/coblox/nectar/pull/46/commits/9d761589ba13e0f001f74ffbad5edec7650d5678.